python/pyarrow/tests/test_csv.py

```diff
@@ -1908,6 +1908,20 @@ def test_write_read_round_trip():
                              parse_options=parse_options)
 
 
+def test_write_quoting_style():
```

Comment:
Can you also expand `test_write_options` on line 324 above?

Reply:
Done